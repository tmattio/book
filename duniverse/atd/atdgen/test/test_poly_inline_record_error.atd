type foo = [ Foo of {x: int} ]
