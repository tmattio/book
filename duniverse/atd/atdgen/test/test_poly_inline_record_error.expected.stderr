File "test_poly_inline_record_error.atd", line 1, characters 13-28:
Inline records are not allowed in polymorphic variants (not valid in OCaml)
